test/fixtures/shared/variables.tf

```diff
@@ -14,23 +14,15 @@
  * limitations under the License.
  */
 
-output "name_example" {
-  description = "Cluster name"
-  value       = "${module.gke.name}"
-}
+variable "project_id" {}
```

Comment:
As per Terraform Standards, all variables must have a description. Especially for ones which tests depend on...